build.gradle

```diff
@@ -13,3 +13,30 @@ allprojects {
         jcenter()
     }
 }
+
+ext {
+    configuration = [
+            applicationId: "org.sinisterstuf.guesstheanimal",
+            buildToolsVersion: "25.0.2",
+            compileSdkVersion: 25,
+            minSdkVersion: 9,
+            targetSdkVersion: 25,
+            versionCodeMajor: 0,
+            versionCodeMinor: 0,
+            versionCodePatch: 1
+    ]
+
+    // Support
+    supportLibraryVersion = "25.3.1"
+
+    // UI
+    butterKnifeVersion = "8.5.1"
+}
+
+def computeVersionCode() {
+    return configuration.versionCodeMajor * 100 + configuration.versionCodeMinor * 10 + configuration.versionCodePatch
+}
+
+def computeVersionName() {
+    return "$configuration.versionCodeMajor.$configuration.versionCodeMinor.$configuration.versionCodePatch"
+}
```

Comment:
For human-editable files please always include a newline at the end of the file, you should be able to set your editor to do this for you, in fact I just added settings for this in master with 2eef933f4a9a9b9cde946e146876f5fb38ae41d3, so a rebase and opening-and-closing-in-IDE should help fix this too.

Reply:
done!